cmd/pkg.go

```diff
@@ -230,6 +272,22 @@ var pkgListCmd = &cobra.Command{
 		fmt.Print(formatted)
 		return nil
 	},
+	Long: `
+The ` + "`list`" + ` command outputs a table that describes all *known* packages (not
+necessarily downloaded, but available from existing registries). This includes
+the following info:
+
+1. Library name
+2. Registry name
+3. Installed status — this is indicated by an asterisk
+
+### Related Commands
+
```

Comment:
ad `ks registry` so they know where to get more installs incase a package is missing.